src/layouts/base-new/index.tsx

```diff
@@ -15,6 +16,12 @@ const { ConsentManager, openConsentManager } = createConsentManager({
   preset: 'oss',
 })
 
+// To be consumed by Sidebar-Sidecar layout where the footer should be attached
+// to the same consent manager, but needs to render within the style of a different layout
+export function FooterWithCookieManager(props: FooterProps) {
+  return <Footer openConsentManager={openConsentManager} {...props} />
+}
```

Comment:
Open to changing where this lives. But given we instantiate the consent manager in this file, I think it makes sense here.

Reply:
oh thats great! I didn't realize